### PR TITLE
Updating logs

### PR DESCRIPTION
**Pull request by b00f**
## Description

Logs should start with the lowercase without period at the end

## Checklist

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] CHANGELOG is updated.

### CODE REVIEW

File: consensus/propose.go [hunk 1]
```diff
@@ -31,19 +31,19 @@ func (s *proposeState) decide() {
 func (s *proposeState) createProposal(height int, round int) {
 	block, err := s.state.ProposeBlock(round)
 	if err != nil {
-		s.logger.Error("We can't propose a block. Why?", "err", err)
+		s.logger.Error("we can't propose a block. Why?", "err", err)
```
Comment (nagarajmanjunath) [inline consensus/propose.go:34]: its should be : `block cannot be proposed`

File: consensus/propose.go [hunk 2]
```diff
@@ -31,19 +31,19 @@ func (s *proposeState) decide() {
...
 		return
 	}
 	if err := s.state.ValidateBlock(block); err != nil {
-		s.logger.Error("Our block is invalid. Why?", "err", err)
+		s.logger.Error("our block is invalid. Why?", "err", err)
```
Comment (nagarajmanjunath) [inline consensus/propose.go:38]: `proposed block is invalid`

File: sync/sync.go [hunk 3]
```diff
@@ -259,7 +259,7 @@ func (sync *synchronizer) Fingerprint() string {
 func (sync *synchronizer) updateBlokchain() {
 	// TODO: write test for me
 	if sync.peerSet.HasAnyOpenSession() {
-		sync.logger.Debug("We have open seasson")
+		sync.logger.Debug("we have open seasson")
```
Comment (nagarajmanjunath) [inline sync/sync.go:262]: `session`

File: sync/sync.go [hunk 4]
```diff
@@ -272,7 +272,7 @@ func (sync *synchronizer) updateBlokchain() {
 			from++
 		}
 
-		sync.logger.Info("We are not synced with the network.")
+		sync.logger.Info("we are not synced with the network.")
```
Comment (nagarajmanjunath) [inline sync/sync.go:275]: `start syncing with network`

**Review by nagarajmanjunath — CHANGES_REQUESTED** [review]

**Review by nagarajmanjunath — APPROVED** [review]

**Review by themantre — APPROVED** [review]

**Pull request merged**